### ISSUE TITLE
Don't caching dist

### ISSUE THREAD

**Issue by AlexPitTech** [isaaceindhoven/composer-velocita#6]
Hello!
I'm trying to use this docker image, but i can't seem to get it to store package dist cache 

```
  velocita:
    container_name: velocita
    image: isaaceindhoven/velocita-proxy
    volumes:
      - ./.cyberplace/velocita:/var/cache/velocita
    restart: always
    environment:
      MIRROR_PACKAGIST_URL: https://repo.packagist.org
      MIRROR_PACKAGIST_TYPE: composer
      MIRROR_ASSET_PACKAGIST_URL: https://asset-packagist.org
      MIRROR_ASSET_PACKAGIST_TYPE: composer
      MIRROR_GITHUB_URL: https://api.github.com
      #MIRROR_GITHUB_TYPE: composer
      #MIRROR_GITHUB_TYPE: git
      MIRROR_GITHUB_TYPE: zip
      VIRTUAL_HOST: velocita.tech
      VIRTUAL_PORT: 8080
      VELOCITA_URL: http://localhost:8080
      VELOCITA_HTTP_PORT: 8080
      VELOCITA_HTTPS_PORT: 8443
```

Also I added target link in my composer.lock

```
            "dist": {
                "type": "zip",
                "url": "http://velocita:8080/mirror/github/repos/doctrine/cache/zipball/382e7f4db9a12dc6c19431743a2b096041bcdd62",
                "reference": "382e7f4db9a12dc6c19431743a2b096041bcdd62",
                "shasum": ""
            },
```

So I can download this zip by url, but don't cache it. Only http responses cached 

How it's work ?

**Comment by gmta** [comment]
Hi @AlexPitTech!

There are a couple of things you need to do to get things working:

* Revert your `composer.lock` to whatever it was before you modified the URL.
* Remove the `MIRROR_GITHUB_*` entries.
* Add a GitHub codeload cache. This is a oneliner that [you can find here](https://github.com/isaaceindhoven/velocita-proxy/blob/7f67e637f2216b30578b7a5b6a0693d24c8f0da7/.env.dist#L17).
* Set the correct `VELOCITA_URL` to (probably) `http://velocita.tech:8080`.
* I'm not sure what `VIRTUAL_HOST` and `VIRTUAL_PORT` are doing in your configuration, but they are not picked up by Velocita - so you can probably remove them.
* Enable the Composer Velocita plugin if you have not already done so, and don't change anything in the way you use `composer`. Just run `composer install/update/require/etc`.

Some more information on the above:

* You must never update your `composer.lock` file for Velocita - just install the `composer-velocita` plugin and install the package as usual by running `composer install` or `composer update`. Check out the plugin's [installation](https://github.com/isaaceindhoven/composer-velocita#installation) and [usage](https://github.com/isaaceindhoven/composer-velocita#usage).
* You are setting `MIRROR_GITHUB_TYPE` to `zip`. That is [not a valid value](https://github.com/isaaceindhoven/velocita-proxy/tree/master/proxy#configuring-mirrors). It could be `simple`, but you are trying to cache the GitHub API which is not a good idea - you should cache GitHub codeload instead!

**Comment by AlexPitTech** [comment]
Thank you for the answer @GMTA 

I use nginxproxy/nginx-proxy on my localhost docker, so velocita.tech is available for host machine. But not available form docker container, where is composer install run. From the container it is available by http://velocita:8080. It is available from velocita container too. I'm not sure which one is correct and I've tried both

So I added MIRROR_GITHUB_CODELOAD_URL=https://codeload.github.com and is is work for me. Thank you!
